core-api/src/main/java/com/optimizely/ab/config/audience/match/ExistsMatch.java

```diff
@@ -20,7 +20,7 @@
 
 import javax.annotation.Nullable;
 
-class ExistsMatch extends AttributeMatch<Object> {
+class ExistsMatch implements Match {
```

Comment:
we don't care about type.  we are matching whether something exists or not.

Reply:
Okay. It surprised me a little since there is a separate declaration of the `”exists”` match type for each separate dimension type that we might need to support, but I agree that that needn’t influence the SDK implementation 👍